api/SpeechRecognitionErrorEvent.json

```diff
@@ -1,173 +0,0 @@
-{
```

Comment:
This file is removed? Why is that?

Reply:
Huh, usually my Git client doesn't have issues removing the correct files during merge conflicts...  I went ahead and added this file back!